### PR TITLE
Complete docstrings

### PR DESCRIPTION
**Pull request by BibMartin**
And here it is.

@ocefpaf 
> Done? Never! Ready for a release? We were probably good enough a few PRs ago :stuck_out_tongue_winking_eye:

I was naively thinking that the work would be finished with `v0.2` ... :wink:

### CODE REVIEW

File: folium/element.py [hunk 1]
```diff
@@ -427,11 +452,21 @@ def __init__(self, data, width="100%", height="100%"):
 
 
 class Div(Figure):
-    """Create a Map with Folium and Leaflet.js.
```
Comment (ocefpaf) [inline folium/element.py]: :scream:

**Comment by ocefpaf** [comment]
Awesome!  Thanks @BibMartin!

I will take a second parse at the open issues/PRs before issuing the new release. Stay tuned...

**Comment by BibMartin** [comment]
Great @ocefpaf ! You're the one that can issue the release (I don't even know what shall be done).

I may come in a few hours with another docs-filling PR, but it can be merged on the release branch I guess.

**Comment by ocefpaf** [comment]
> I don't even know what shall be done.

Nothing fancy.  I just tag it on GitHub and upload the source dist to PyPI.  There are projects with way more complex steps, but for now ours is still simple.  We will add some complexity for publishing the docs though.

> I may come in a few hours with another docs-filling PR, but it can be merged on the release branch I guess.

OK.  I won't create a `v0.2.0` branch until we are all set to avoid confusion.  We can keep pointing PRs to master.

**Pull request merged**